components/admin_console/group_settings/groups_list/groups_list.test.tsx

```diff
@@ -4,19 +4,37 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
-import GroupsList from 'components/admin_console/group_settings/groups_list/groups_list.jsx';
+import GroupsList from 'components/admin_console/group_settings/groups_list/groups_list';
 
 describe('components/admin_console/group_settings/GroupsList', () => {
+    type State = {
+        checked: any;
+        loading: boolean;
+        page: number;
+        showFilters: boolean;
+        searchString: string;
+        filterIsConfigured?: boolean;
+        filterIsUnconfigured?: boolean;
+        filterIsLinked?: boolean;
+        filterIsUnlinked?: boolean;
+    }
+
+    const baseProps = {
+        groups: [
+            {primary_key: 'test1', name: 'test1', mattermost_group_id: null, has_syncables: null},
+            {primary_key: 'test2', name: 'test2', mattermost_group_id: 'group-id-1', has_syncables: false},
+        ],
+        total: 2,
+        actions: {
+            getLdapGroups: jest.fn().mockReturnValue(Promise.resolve()),
+            link: jest.fn(),
+            unlink: jest.fn(),
+        }
+    };
     test('should match snapshot, while loading', () => {
         const wrapper = shallow(
             <GroupsList
-                groups={[]}
```

Comment:
You're changing the test here, please revert so that the test behaviour remains the same.

Reply:
Reverted to original, had to specify type in shallow<GroupsList>, hope that is okay.